ios/RNLinksdk.m

```diff
@@ -39,7 +39,8 @@
 @interface RNLinksdk ()
 @property (nonatomic, strong) id<PLKHandler> linkHandler;
 @property (nonatomic, strong) UIViewController* presentingViewController;
-@property (nonatomic, strong) RCTResponseSenderBlock completionCallback;
```

Comment:
completionCallback gets split up into successCallback and exitCallback, and called in the respective ObjC onSuccess and onExit methods